hoomd/md/PotentialExternal.h

```diff
@@ -4,6 +4,7 @@
 #include "hoomd/ForceCompute.h"
 #include "hoomd/GPUArray.h"
 #include "hoomd/GlobalArray.h"
+#include "hoomd/managed_allocator.h"
```

Comment:
Why was this line added? The managed allocator class doesn't appear in the diff for this file.

Reply:
Functions from `managed_allocator` are used in the file (`make_managed_shared`), but `managed_allocator.h` was previously included only in files before `PotentialExternal.h` in other files. In the current code, remove this include and the compilation will fail with an error.